### ISSUE TITLE
 'ClientResponse' object has no attribute 'get header'

### ISSUE THREAD

**Issue by mnasiri** [tomplus/kubernetes_asyncio#122]
### This issue is seen when run my project with ver.12.0.0 of  kubernetes_asyncio which it was worked with ver. 11.3.0.
```
resp = await v1.read_namespaced_pod_log(podname, namespace, container=container, **params)
  File "..\.venv\lib\site-packages\kubernetes_asyncio\client\api_client.py", line 191, in __call_api
    content_type = response_data.getheader('content-type')
AttributeError: 'ClientResponse' object has no attribute 'getheader'
```

https://github.com/tomplus/kubernetes_asyncio/blob/6502fde25fbd0abe6f7ab6dce25bc5edd04f1126/kubernetes_asyncio/client/api_client.py#L191 

to fix it, I change it to 
`     content_type = response_data.headers.get('content-type')`

I use : 
aiohttp==3.6.3 
urllib3==1.25.11
Thanks for your attempts.

**Comment by tomplus** [comment]
@mnasiri Thanks for reporting this issue.

It appears when you call any methods with preload_content=False (eg. Watch() works in this way). I'll fix it ASAP.

**Comment by tomplus** [comment]
It has been fixed in v12.0.1
